src/engine/world/skill-util/harvest-skill.ts

```diff
@@ -166,10 +180,18 @@ export function handleHarvesting(details: ObjectInteractionAction, tool: Harvest
                     details.player.skills.addExp(skill, target.experience);
                     if (randomBetween(0, 100) <= target.break) {
                         details.player.playSound(soundIds.oreDepeleted);
-                        details.player.instance.replaceGameObject(target.objects.get(details.object.objectId),
-                            details.object, randomBetween(target.respawnLow, target.respawnHigh));
+
+                        const replacementObject = target.objects.get(details.object.objectId);
+                        const respawnTime = randomBetween(target.respawnLow, target.respawnHigh);
+
+                        if (replacementObject) {
+                            details.player.instance.replaceGameObject(replacementObject, details.object, respawnTime);
+                        }
+
+                        // TODO (Jameskmonger) cancel animation
+                        // details.player.playAnimation(null);
```

Comment:
Is there a reason this is marked as a TODO and isn't just called?

Reply:
my mistake, I didn't realise `playAnimation` could accept a `null`. I will change now